crates/wash/src/cli/config.rs

```diff
@@ -84,6 +116,221 @@ impl CliCommand for ConfigCommand {
+                    ));
+                }
+
+                warn!(
+                    "Found {} files for cleanup. Files to be deleted:\n{}\n\nDo you want to proceed with the deletion? (y/N)",
+                    cleanup_files.len(),
+                    files_summary
+                );
+
+                let mut successful_deletions = 0;
+                let mut failed_paths = Vec::new();
+
+                let mut confirmation = String::new();
+                io::stdin().read_line(&mut confirmation)?;
+
+                if !confirmation.trim().eq_ignore_ascii_case("y") {
+                    return Ok(CommandOutput::ok(
+                        format!("Skipped deletion of {} files", cleanup_files.len()),
+                        Some(serde_json::json!({
+                            "message": "File deletion skipped.",
+                            "success": true,
+                        })),
+                    ));
+                }
+
+                for path in &cleanup_files {
+                    match std::fs::remove_file(path) {
+                        Ok(_) => {
+                            info!("Successfully deleted file: {:?}", path.display());
+                            successful_deletions += 1
+                        }
+                        Err(e) => {
+                            error!("Failed to delete {} file: {}", path.display(), e);
+                            failed_paths.push(path.clone());
+                        }
+                    }
+                }
+
+                for path in &cleanup_dirs {
+                    match std::fs::remove_dir_all(path) {
+                        Ok(_) => {
+                            info!("Successfully deleted dir: {:?}", path.display());
+                        }
+                        Err(e) => {
+                            error!("Failed to delete dir {}: {}", path.display(), e);
+                            failed_paths.push(path.clone());
+                        }
+                    }
+                }
+
+                if !failed_paths.is_empty() {
+                    return Ok(CommandOutput::error(
+                        format!("Failed to delete {} files", failed_paths.len()),
+                        Some(serde_json::json!({
+                            "message": format!("Partial failure: Deleted {}/{} files.",
+                              successful_deletions,
+                              cleanup_files.len()),
+                            "deleted": successful_deletions,
+                            "failed_count": failed_paths.len(),
+                            "success": false,
+                        })),
+                    ));
+                }
+
+                return Ok(CommandOutput::ok(
+                    format!("Successfully deleted {} files", successful_deletions),
+                    Some(serde_json::json!({
+                        "message": format!("{} files deleted successfully.",successful_deletions),
```

Comment:
```suggestion
                    format!("Successfully deleted {successful_deletions} files"),
                    Some(serde_json::json!({
                        "message": format!("{successful_deletions} files deleted successfully."),
```

This format better matches the rest of the crate's usage